src/main/resources/.env.example

```
@@ -23,4 +23,45 @@ LOG_LEVEL_SPRING=WARN              # 필요한 경고만 로깅
 LOG_LEVEL_SQL=ERROR                # 에러만 로깅
 
 # p6spy (비활성화)
-P6_SPY=false                       # 성능 영향 방지
\ No newline at end of file
+P6_SPY=false                       # 성능 영향 방지
+
+# mail
+RECEIVER_EMAIL_ADDRESS=example@gmail.com    #test mail 주소
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)